<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>구글 한줄 번역기</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_kor_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>한국어 입력</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_eng_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;border-width:2px;border-color:rgb(0, 0, 255)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_jpn_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;border-width:2px;border-color:rgb(0, 0, 255)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_chn_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;border-width:2px;border-color:rgb(0, 0, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>영어</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>일본어</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>중국어</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_trans">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>80</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;border-width:2px;border-color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>번역</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
